ui/components/ui/form-field/form-field.js

```diff
@@ -132,40 +132,44 @@ export default function FormField({
           />
         )}
         {error && (
-          <Typography
+          <Text
             color={TextColor.errorDefault}
-            variant={TypographyVariant.H7}
+            variant={TextVariant.bodySm}
+            as="h6"
             className="form-field__error"
           >
             {error}
-          </Typography>
+          </Text>
         )}
         {warning && (
-          <Typography
+          <Text
             color={TextColor.textAlternative}
-            variant={TypographyVariant.H7}
+            variant={TextVariant.bodySm}
+            as="h6"
             className="form-field__warning"
           >
             {warning}
-          </Typography>
+          </Text>
         )}
         {passwordStrength && (
-          <Typography
+          <Text
             color={TextColor.textDefault}
-            variant={TypographyVariant.H7}
+            variant={TextVariant.bodySm}
+            as="h6"
             className="form-field__password-strength"
           >
             {passwordStrength}
-          </Typography>
+          </Text>
         )}
         {passwordStrengthText && (
-          <Typography
+          <Text
             color={TextColor.textAlternative}
-            variant={TypographyVariant.H8}
+            variant={TextVariant.bodyXs}
+            as="h6"
             className="form-field__password-strength-text"
           >
             {passwordStrengthText}
-          </Typography>
+          </Text>
         )}
       </Box>
     </div>
```

Comment:
Adding screenshots for FormField

### Before
<img width="1440" alt="Screenshot 2023-04-29 at 8 43 10 AM" src="https://user-images.githubusercontent.com/8112138/235268347-362e1267-533e-4570-8ed6-7da650e05cb0.png">

### After
<img width="1440" alt="Screenshot 2023-04-29 at 8 41 40 AM" src="https://user-images.githubusercontent.com/8112138/235268341-c7d15b13-f3b1-4dd3-a314-80fdf9a2c727.png">